js/utils/follow.js

```diff
@@ -0,0 +1,49 @@
+import $ from 'jquery';
+import app from '../app';
+import Dialog from '../views/modals/Dialog';
+
+export default {
```

Comment:
We can also export each function individually as opposed to exporting one big object with nested functions. It's arguably a cleaner syntax that saves you a level of indentation. But, I suppose it's a personal preference thing as well.